### PR TITLE
Add descriptor for the build-tool-detector service

### PR DESCRIPTION
**Pull request by sbose78**
depends on 
- https://github.com/fabric8-services/build-tool-detector/pull/7
- quay repo creation as tracked in https://gitlab.cee.redhat.com/dtsd/housekeeping/issues/2418

### CODE REVIEW

**Comment by riuvshin** [comment]
[test]

File: dsaas-services/build-tool-detector.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+services:
+- hash: 8b49875a886a1fb2a290afb70121edac6eb2b321
```
Comment (riuvshin) [inline dsaas-services/build-tool-detector.yaml]: please bump version to recent one `3ea0b04` that exists on quay to fix ci check failure

Comment (aditya-konarde) [reply dsaas-services/build-tool-detector.yaml]: @sbose78

Comment (sbose78) [reply dsaas-services/build-tool-detector.yaml]: Done :)

**Comment by sbose78** [comment]
[test]

**Comment by aditya-konarde** [comment]
[test]

**Comment by sbose78** [comment]
Done :)

**Comment by sbose78** [comment]
This being a new service - CI test failures aren't relevant.

**Pull request merged**